nservicebus/operations/auditing.md

```diff
@@ -71,3 +71,15 @@ If no OverrideTimeToBeReceived is defined then:
 ## Configuring auditing
 
 partial: configuration
+
+## Filtering audit messages
+
+By default all messages are audited, to control what message types that gets audited see our [audit filter sample](/nservicebus/audit-filter/).
```

Comment:
could use some little tweaks. How about:
By default all messages are audited. To control which message types are audit, see the [audit filter sample]

Reply:
That link does not point to a sample